### ISSUE TITLE
Review documentation classification

### ISSUE THREAD

**Issue by astrojuanlu** [poliastro/poliastro#975]
According to [Divio classification for documentation](https://documentation.divio.com/), there are four types of it:

- Tutorials
- How-to Guides
- Explanations
- References

We should review our docs to separate these categories, and we will probably find some gaps :)

**Comment by jorgepiloto** [comment]
I would like to solve for this issue. Several questions:

* Do you think we should also separate internal directories according to this classification?
* I came up with the following layout. However, I am not sure how to include the `Explanations` section or if some of the topics presented here might be included in that category...


```bash
.
└── Welcome to poliastro
│   ├── About poliastro
│   └── Whats new
├── How to
│   ├── Getting started
│   ├── Use poliastro
│   └── Contribute
├── Tutorials
│   └── Gallery
└── References
    ├── High level API
    ├── Core level API
    └── Bibliography

**Comment by astrojuanlu** [comment]
> Do you think we should also separate internal directories according to this classification?

I don't think we have a large number of files, so perhaps we can live with a flat hierarchy for now? However I don't have a strong opinion on this.

> I came up with the following layout.

I like it very much!

> However, I am not sure how to include the Explanations section or if some of the topics presented here might be included in that category...

Theoretical knowledge + Understanding-oriented = https://github.com/astrobook/astrobook ? :slightly_smiling_face: 

In other words: I think the "explanations" here refers to what textbooks and manuals usually cover. So, until you finish your open source book, probably there's not much material we can add here :)

**Comment by jorgepiloto** [comment]
I agree with your last comment! Pull request coming soon... :rocket:

**Comment by astrojuanlu** [comment]
Addressed in #989 :rocket:
